### ISSUE TITLE
New Function: cave api-test

### ISSUE THREAD

**Issue by connor-makowski** [MIT-CAVE/cave_cli#27]
Add support for api testing. See: https://github.com/MIT-CAVE/cave_app/blob/0.1.0/cave_api/README.md#testing

I am not happy with the name and would like a more general process if possible. Feel free to take some liberties as you see fit.

**Comment by mrkwanzaa** [comment]
I am going to call this `cave test` if that works. I'll update the readme in cave_app once this is merged into main.
